src/os/bluestore/BlueFS.cc

```diff
@@ -168,6 +168,7 @@ class BlueFS::SocketHook : public AdminSocketHook {
 		f->dump_int(("dev-"+to_string(i)).c_str(), sizes[i]);
 	    }
           }
+          f->dump_int("size", r.second->fnode.size);
```

Comment:
Shouldn't we go forward and dump ino and mtime as well?

Reply:
Does `ino` make sense? We do not see it until we enable logging, and when we enable logging, we also see it on open().
I have never used mtime.

But if you think it is worth, I will add them too.

Reply:
Who knows what one might need during troubleshooting... And it's almost for free, so why not having that?